en/docs/learn/configuring-just-in-time-provisioning-for-an-identity-provider.md

```diff
@@ -100,7 +100,23 @@ below:
             store
             configuration](../../setup/configuring-user-stores)
             .
-        
+
+        !!! note
+
+            To associate any user having the same JIT provisioned username with the local user having the same username, 
+            add the following configuration to `<IS-HOME>/repository/conf/deployment.toml`.
+
+            ``` toml
+            [authentication]
+            jit_provisioning.associating_to_existing_user = true
+            ```
+
+            Note that if this is enabled, there's a risk of unauthorized profile update.
+
+            This configuration is available as an update in WSO2 IS 5.11.0 from update level 69 onwards 
+            (Updates 2.0 model). If you don't already have this update, see the instructions on 
+            [updating your product](https://updates.docs.wso2.com/en/latest/updates/overview/).
+            Previous update levels have this behavior by default.
```

Comment:
Do we need this line? Since we say that the update is available from update level 69 onwards?

Reply:
Addressed with commit 3ab0f98903f9420d8a3dac66a629df96149596b8